doc/mgr/smb.rst

```diff
@@ -179,6 +179,55 @@ subvolume
 readonly
     Creates a read-only share
 
+Update Share QoS
+++++++++++++++++
+
+.. prompt:: bash #
+
+   ceph smb share update cephfs qos <cluster_id> <share_id> [--read-iops-limit=<int>] [--write-iops-limit=<int>] [--read-bw-limit=<int>] [--write-bw-limit=<int>]
+
+Update Quality of Service (QoS) settings for a CephFS-backed share. This allows administrators to apply per-share rate limits on SMB input/output (I/O) operations, specifically limits on IOPS (Input/Output Operations per Second) and bandwidth (in bytes per second) for both read and write operations.
+
+Options:
+
+read_iops_limit
+    Optional integer. Maximum number of read operations per second (0 = disabled)
+write_iops_limit
+    Optional integer. Maximum number of write operations per second (0 = disabled)
+read_bw_limit
+    Optional integer. Maximum allowed bandwidth for read operations in bytes per second (0 = disabled)
+write_bw_limit
+    Optional integer. Maximum allowed bandwidth for write operations in bytes per second (0 = disabled)
+
+Behavior:
```

Comment:
Please ignore if this was intentional.

```suggestion
Behaviour:
```

Reply:
I don't mind, it's just I see more `Behavior` in most of the .rst files

Reply:
Historically, a lot of the original Ceph team were American (IIRC Inktank was originally based in CA). I don't know if there's a standard across all the documentation files to use American spelling vs. UK/commonwealth spelling. Maybe it's more like Wikipedia where each page is supposed to be consistent but it varies across articles.  :-)